src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
`null`ing `originalEvent` like @gabrielschulhof suggests fixes the test: https://jsfiddle.net/74eutos8/1/embedded/result/